### PR TITLE
add docs for ingress/service and overall docs layout

### PR DESCRIPTION
**Pull request by M00nF1sh**
add docs for ingress/service and overall docs layout

preview: https://m00nf1sh.github.io/aws-alb-ingress-controller/

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## AWS LoadBalancer Controller
+
+AWS LoadBalancer Controller is a controller to help manage Elastic Load Balancers for a Kubernetes cluster.
+
+  - It satisfies Kubernetes [Ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/) by provisioning [Application Load Balancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
+  - It satisfies Kubernetes [Service resources](https://kubernetes.io/docs/concepts/services-networking/service/) by provisioning
+[Network Load Balancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html).
+
+This project was priorly known as "AWS ALB Ingress Controller", we rebranded it to be "AWS LoadBalancer Controller".
```
Comment (kishorj) [inline docs/README.md:13]: formerly instead of priorly sounds better to me

File: docs/guide/service/nlb_ip_mode.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# NLB IP mode
+AWS Load Balancer Controller supports Network Load Balancer (NLB) with IP targets for pods running on Amazon EC2 instances and AWS Fargate through Kubernetes service of type `LoadBalancer` with proper annotation. In this mode, the AWS NLB targets traffic directly to the Kubernetes pods behind the service, eliminating the need for an extra network hop through the worker nodes in the Kubernetes cluster.
+
+## Prerequisites
+* AWS Loadbalancer Controller >= v2.0.0
+* Kubernetes >= v1.14 for Service type NodePort.
+* Kubernetes >= v1.20 for Service type Loadbalancer.
```
Comment (kishorj) [inline docs/guide/service/nlb_ip_mode.md:7]: LoadBalancer

File: docs/README.md (hunk 1 above)
Comment (M00nF1sh) [reply docs/README.md:13]: nice :D

File: docs/guide/service/nlb_ip_mode.md (hunk 2 above)
Comment (M00nF1sh) [reply docs/guide/service/nlb_ip_mode.md:7]: sure

**Review by kishorj — APPROVED** [review]

**Pull request merged**